UI/window-basic-main.cpp

```diff
@@ -4466,9 +4466,7 @@ void OBSBasic::on_scenes_currentItemChanged(QListWidgetItem *current,
 	obs_source_t *source = NULL;
 
 	if (current) {
-		obs_scene_t *scene;
-
-		scene = GetOBSRef<OBSScene>(current);
+		obs_scene_t *scene = GetOBSRef<OBSScene>(current);
```

Comment:
This change is no longer required.

```suggestion
		obs_scene_t *scene;

		scene = GetOBSRef<OBSScene>(current);
```